L1Trigger/L1TMuonOverlapPhase2/plugins/L1TMuonOverlapPhase2TrackProducer.h

```diff
@@ -0,0 +1,43 @@
+#ifndef OMTFProducer_H
+#define OMTFProducer_H
+
+#include "FWCore/Framework/interface/one/EDProducer.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+
+#include "L1Trigger/L1TMuonOverlapPhase2/interface/OmtfEmulation.h"
+#include "SimDataFormats/Track/interface/SimTrackContainer.h"
+
+class L1TMuonOverlapPhase2TrackProducer : public edm::one::EDProducer<edm::one::WatchRuns> {
+public:
+  L1TMuonOverlapPhase2TrackProducer(const edm::ParameterSet&);
+
+  ~L1TMuonOverlapPhase2TrackProducer() override;
+
+  void beginJob() override;
+
+  void endJob() override;
+
+  void beginRun(edm::Run const& run, edm::EventSetup const& iSetup) override;
+
+  void endRun(edm::Run const& run, edm::EventSetup const& iSetup) override{};
```

Comment:
```suggestion
```

Reply:
It cannot be removed as this method is purely virtual in base class.